### ISSUE TITLE
Missing Feature : A preference pane to show and edit all per-site shield settings

### ISSUE THREAD

**Issue by alexwykoff** [brave/browser-laptop#3611]
**Did you search for similar issues before submitting this one?**
Yes

**Describe the issue you encountered:**
When visiting www.twitter.com/brianbondy and setting the shield value to 'Block scripts', www.twitter.com recognizes that scripts have been blocked and redirects the user to mobile.twitter.com. Shield values are not global to the domain+tld so it recognizes the subdomain as a new site. The redirect is quick so the user has no means to recover if they wish to unblock scripts.

**Expected behavior:**
The user needs a means to edit values in these situations so they can restore prior conditions.

- Platform (Win7, 8, 10? macOS? Linux distro?):
Tested on OS X

- Brave Version:
Discovered in 0.11.6 rc2, but has been around for as long as we have had shields.

- Steps to reproduce:
    1. Visit www.twitter.com/brianbondy
    2. Set the shield value of 'Block Scripts' to on
    3. Attempt to un-set that value

- Screenshot if needed:

- Any related issues:

**Comment by bbondy** [comment]
@bradleyrichter I think you have an issue for this already, could you link to it. @alexwykoff once it's duped to that you could add your particular STR for a use case to check when testing it.

**Comment by bradleyrichter** [comment]
I think this will need to be a 1.1 feature.

**Comment by bbondy** [comment]
pls put your mockup here or dupe to another @bradleyrichter ya I'm going to move it for now.

**Comment by diracdeltas** [comment]
this seems to be a duplicate of the issue that was fixed by f4bcc0813f8bab3561a5032450071b5df89e400e in 0.11.3
